README.md

```diff
@@ -79,6 +79,31 @@ If defined, the style-loader will re-use a single `<style>` element, instead of
 
 If convertToAbsoluteUrls and sourceMaps are both enabled, relative urls will be converted to absolute urls right before the css is injected into the page. This resolves [an issue](https://github.com/webpack/style-loader/pull/96) where relative resources fail to load when source maps are enabled.  You can enable it with the convertToAbsoluteUrls query parameter (`?convertToAbsoluteUrls`).
 
+#### `onCssLoad`
+
+If an `onCssLoad` function is defined, this function will run every time a css is loaded into the page through the style-loader.
+The `onCssLoad` function gets one parameter that is the original css that the style-loder is loading into the page.
+Possible return values of `onCssLoad`:
+* **nothing** - if nothing (undefined) is returned from the `onCssLoad` function, the original css will be loaded without any changes.

```

Comment:
@michael-ciniawsky The `nothing`/`undefined` option is a convenience. The logic behind it is that if the user wants to write some code that runs on css load but doesn't change the css (a hook, not a transformation) I won't force him to return the original css. Since this seems complicated (more than it actually is) I will remove this option to keep it simple.